cache/cache.cpp

```diff
@@ -701,8 +699,12 @@ int dnet_cmd_cache_io(struct dnet_backend *backend,
 }
 
 static uint64_t n2_request_get_io_flags(const n2_request &request) {
-	switch (request.cmd.cmd) {
+	using namespace ioremap::elliptics;
+
 	// TODO: Get ioflags from all requests that contain ioflags field
+	switch (request.cmd.cmd) {
+	case DNET_CMD_DEL_NEW:
+		return std::static_pointer_cast<n2::remove_request>(request.body)->ioflags;
```

Comment:
(not mandatory, performance notes) At this line temporary `shared_ptr` object is created. If we need to only reinterpret underlying object, better to do `static_cast<const T *>(p.get())` or  `static_cast<const T &>(*p)`.